app/src/main/java/org/jellyfin/androidtv/util/KeyProcessor.java

```diff
@@ -420,47 +418,8 @@ public void onError(Exception exception) {
                     toggleFavorite(false);
                     return true;
                 case MENU_MARK_PLAYED:
-                    if (currentItemIsFolder) {
-                        // confirm
-                        new AlertDialog.Builder(mCurrentActivity)
-                                .setTitle(R.string.lbl_mark_played)
-                                .setMessage(mCurrentActivity.getString(R.string.lbl_confirm_mark_watched))
-                                .setNegativeButton(mCurrentActivity.getString(R.string.lbl_no), new DialogInterface.OnClickListener() {
-                                    @Override
-                                    public void onClick(DialogInterface dialog, int which) {
-
-                                    }
-                                }).setPositiveButton(mCurrentActivity.getString(R.string.lbl_yes), new DialogInterface.OnClickListener() {
-                            @Override
-                            public void onClick(DialogInterface dialog, int which) {
-                                    markPlayed();
-                            }
-                        }).show();
-
-                    } else {
-                        markPlayed();
-                    }
-                    return true;
                 case MENU_UNMARK_PLAYED:
```

Comment:
Should this case not be calling `markUnplayed()`?

Reply:
Yup, I misread it.